src/open_inwoner/accounts/models.py

```diff
@@ -429,6 +418,19 @@ def get_logout_url(self) -> str:
                 return reverse("eherkenning_oidc:logout")
             return reverse("logout")
 
+    def has_usable_email(self) -> bool:
+        return User.is_usable_email(self.email)
```

Comment:
That's a bit odd semantically. Perhaps `return User._has_usable_email(self.email)` or `return User.check_email(self.email)`? Not a big deal though, since we're always calling `has_usable_email` on the instance.